utils/kata-manager.sh

```diff
@@ -155,9 +161,18 @@ github_get_release_file_url()
 
 	local download_url
 
+	#parses the json response from github. Python is on every machine out there, but not jq.
+	local python_jq="import json
+import sys
+response=json.loads(sys.stdin.read())
+version=sys.argv[1]
+for tag in response:
+    if tag['tag_name']==version:
+        for file in tag['assets']:
+            print(file['browser_download_url'])"
+
 	download_url=$(curl -sL "$url" |\
-		jq --arg version "$version" \
-		-r '.[] | select(.tag_name == $version) | .assets[].browser_download_url' |\
+		python -c "$python_jq", $version|\
```

Comment:
- Consistency: above, we use `python3`, not `python`.
- As above, `/usr/bin/python` might be python2 or python3. On Fedora 37 it's python3.

Reply:
Good catch on the consistency. To the best of my knowledge, the script will work on both python2 and 3